testdata/go/v4/memcached-operator/README.md

```diff
@@ -68,27 +68,48 @@ make undeploy
 
 ## Project Distribution
 
-Following are the steps to build the installer and distribute this project to users.
+Following the options to release and provide this solution to the users.
+
+### By providing a bundle with all YAML files
 
 1. Build the installer for the image built and published in the registry:
 
 ```sh
 make build-installer IMG=<some-registry>/memcached-operator:tag
 ```
 
-NOTE: The makefile target mentioned above generates an 'install.yaml'
+**NOTE:** The makefile target mentioned above generates an 'install.yaml'
 file in the dist directory. This file contains all the resources built
-with Kustomize, which are necessary to install this project without
-its dependencies.
+with Kustomize, which are necessary to install this project without its
+dependencies.
 
 2. Using the installer
 
-Users can just run kubectl apply -f <URL for YAML BUNDLE> to install the project, i.e.:
+Users can just run 'kubectl apply -f <URL for YAML BUNDLE>' to install
+the project, i.e.:
 
 ```sh
 kubectl apply -f https://raw.githubusercontent.com/<org>/memcached-operator/<tag or branch>/dist/install.yaml
 ```
 
+### By providing a Helm Chart
+
+1. Build the chart using the optional helm plugin
+
+```sh
+operator-sdk edit --plugins=helm/v1-alpha
```

Comment:
@acornett21 we need to check if we have the helm plugin or if we want here.
If not we can add the helm plugin or remove it from the README.